jerry-core/ecma/base/ecma-helpers-string.c

```diff
@@ -198,6 +198,26 @@ ecma_new_ecma_string_from_magic_string_ex_id (lit_magic_string_ex_id_t id) /**<
   return string_desc_p;
 } /* ecma_new_ecma_string_from_magic_string_ex_id */
 
+#ifndef CONFIG_DISABLE_ES2015_SYMBOL_BUILTIN
+/**
+ * Allocate new ecma-string and fill it with reference to the symbol descriptor
+ *
+ * @return pointer to ecma-string descriptor
+ */
+ecma_string_t *
+ecma_new_symbol_from_descriptor_string (ecma_value_t string_desc) /**< ecma-string */
+{
+  JERRY_ASSERT (!ecma_is_value_symbol (string_desc));
+
+  ecma_string_t *symbol_p = ecma_alloc_string ();
+  symbol_p->refs_and_container = ECMA_STRING_REF_ONE | ECMA_STRING_CONTAINER_HEAP_UTF8_SYMBOL;
+  symbol_p->u.symbol_descriptor = string_desc;
+  symbol_p->hash = (uint16_t) (((uintptr_t) symbol_p) >> 3);
```

Comment:
I think we discussed 1 bit for built-ins. Is this changed? Also a define would be good for this constant.